src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/SynthesizedClosureEnvironment.cs

```diff
@@ -89,13 +98,20 @@ private static void AssertIsClosureScopeSyntax(SyntaxNode syntaxOpt)
 
         public override ImmutableArray<Symbol> GetMembers()
         {
-            var members = base.GetMembers();
+            var builder = ArrayBuilder<Symbol>.GetInstance();
+            builder.AddRange(base.GetMembers());
```

Comment:
Sounds reasonable. Thank you!

---
In reply to: [134861344](https://github.com/dotnet/roslyn/pull/21664#discussion_r134861344) [](ancestors = 134861344)